open-company/open-company-web
# Mobile NUX

Card: https://trello.com/c/NCcZ4AXu

Check against the screens in the latest version of the master sketch file.
*Note 1*: the copy of the onboard screens and tooltips are not the same, i kept the one we have for desktop.
*Note 2*: the 4th onboard tooltip is not visible on mobile since we don't have the invite people button.

To test:
- logout of Carrot on your mobile device browser
- visit our homepage
- tap on the hamburger menu (top left corner)
- tap on Get Started for Free ->
- tap on the blue back arrow at the top left corner
- [ ] are you back to the homepage? Good
- [ ] is the hamburger menu closed? Good
- tap again on Get started for free ->
- [ ] fill in your email, do you see the email keyboard? Good
- fill in a password
- [ ] do you NOT see the blue back arrow in the first/last name collection screen? Good
- fill in your first and last name
- [ ] use the FileStack picker to add a profile picture, did it work? Good
- continue
- [ ] use the FileStack picker to add an avatar for your organization, did it work? Good
- add a name for your org
- continue
- [ ] do you see the blue setup screen? Good (remember, you should not see it again ever again for this user)
- [ ] do you see the first onboard screen? Good
- tap the bottom button
- [ ] do you see the create post screen? Good
- [ ] can you NOT edit the post board, headline, body and topic? Good
- tap Post
- [ ] do you see the 1 of 3 tooltip pointing to your new post? Good
- tap Cool
- [ ]  do you see the 2 of 3 tooltip? Good
- tap Got It
- [ ]  do you see the 3 of 3 tooltip? Good
- tap Makes sense
- [ ] do you see the last onboard screen? Good
- tap on Start using Carrot
- [ ] are you back to the board screen? Good
- [ ] refresh the page, do you NOT see the blue setup screen but only the grey waving carrot while loading? Good
